src/main/java/propertyFiles/Bundle.properties

```diff
@@ -1934,7 +1934,7 @@ file.downloadBtn.format.all=All File Formats + Information
 file.downloadBtn.format.tab=Tab-Delimited
 file.downloadBtn.format.original={0} (Original File Format)
 file.downloadBtn.format.rdata=RData
-file.downloadBtn.format.var=Variable Metadata
+file.downloadBtn.format.var=DDI Codebook v2
 file.downloadBtn.format.citation=Data File Citation
 file.download.filetype.unknown=Original File Format
 file.more.information.link=Link to more file information for 
```

Comment:
I'm putting this here at the end... code coverage is ever so slightly down. At our last monthly meeting we talked about trying to move it up. Maybe we should add tests.

Reply:
I started adding something to the DDI export test; but then stopped myself, seeing how the issue was a "3", and I was also told to focus on something urgent-ish r/n.

Reply:
Ok. Just doing my due diligence. Resolving. 😄